packages/server/src/app.ts

```diff
@@ -109,10 +110,19 @@ function errorHandler(err: any, req: Request, res: Response, next: NextFunction)
     sendOutcome(res, badRequest('File too large'));
     return;
   }
-  logger.error('Unhandled error', err);
+  try {
+    getRequestContext().logger.error('Unhandled error', err);
+  } catch {
+    globalLogger.error('Unhandled error', err);
+  }
   res.status(500).json({ msg: 'Internal Server Error' });
 }
 
+async function attachRequestContext(req: Request, res: Response, next: NextFunction): Promise<void> {
```

Comment:
Could this be moved into `middleware.ts` so all of the context code is in a single place?